frontend/src/metabase/query_builder/components/NativeQueryEditor.styled.tsx

```diff
@@ -90,3 +91,34 @@ export const NativeQueryEditorRoot = styled.div`
     background-color: ${color("bg-light")};
   }
 `;
+
+export const aceEditorStyles = css`
```

Comment:
The popover is displayed outside of the element in the document.body; we cannot have these styles in `NativeQueryEditorRoot`